pulpcore/pulpcore/app/viewsets/custom_filters.py

```diff
@@ -60,3 +63,99 @@ class IsoDateTimeFilter(DateTimeFilter):
     def __init__(self, *args, **kwargs):
         kwargs.setdefault('help_text', _('ISO 8601 formatted dates are supported'))
         super().__init__(*args, **kwargs)
+
+
+class ContentRepositoryVersionFilter(Filter):
```

Comment:
I want to DRY this but when I tried, it wasn't passing the "value" correctly.  I'll get to it later if I can.